src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/DeleteDescriptorsValidation.feature.disabled

```diff
@@ -0,0 +1,87 @@
+             Then it should respond with 204
+
+        @ignore
+        Scenario: 02 Verify error handling when deleting a descriptor using a invalid id
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/00112233445566"
+             Then it should respond with 404
+              And the response body is
+                  """
+                  {
+                      "detail": "Data validation failed. See 'validationErrors' for details.",
+                      "type": "urn:ed-fi:api:bad-request:data-validation-failed",
+                      "title": "Data Validation Failed",
+                      "status": 404,
+                      "correlationId": null,
+                      "validationErrors": {
+                          "$.id": [
+                              "The value '00112233445566' is not valid."
+                          ]
+                      }
+                  }
+                  """
+
+        @ignore
+        Scenario: 03 Verify error handling when trying to delete an item that has already been deleted
+            # The id value should be replaced with the resource created in the Background section
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 204
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 404
+
+        @ignore
+        Scenario: 04 Verify response code when GET a deleted resource
+            # The id value should be replaced with the resource created in the Background section
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 204
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 404
+
+        @ignore
+        Scenario: 05 Ensure clients can delete a descriptor that is not beign used by any resource
+            Given the system has this "disabilityDescriptors" descriptor
+                  | codeValue       | namespace                            | shortDescription     |
+                  | Disability Test | uri://ed-fi.org/DisabilityDescriptor | User Disability Test |
+             When a DELETE request is made to "/ed-fi/disabilityDescriptors/{id}"
+             Then it should respond with 204
+
+        @ignore
+        Scenario: 06 Ensure clients cannot delete a descriptor that is being used by other Resources
+            Given the system has this "educationOrganizationCategoryDescriptors" descriptor
+                  | codeValue              | namespace                                               | shortDescription       |
+                  | Local Education Agency | uri://ed-fi.org/EducationOrganizationCategoryDescriptor | Local Education Agency |
```

Comment:
This scenario will not work yet, because there is nothing that depends on this descriptor. You need to create a school (for example) that uses this descriptor. Then on line 77 the error message should mention School instead of EducationOrganizationCategory.